### ISSUE TITLE
v35 regression: Metabase giving error message when null values are returned from Custom SQL Questions

### ISSUE THREAD

**Issue by exabrial** [metabase/metabase#12232]
**Describe the bug**
Metabase is giving an error message `Cannot read property 'length' of null` when a null value is returned from a Custom SQL Query. 

**Logs**
No errors appear in the logs

**To Reproduce**
Create a bar graph visualization with labels from this dataset returned from the database. The first column is a varchar, the second is a signed integer:

| minute_interval | counted |
|-----------------|---------|
| 14:37           |`<null>`|
| 14:38           |`<null>`|
| 14:39           |`<null>`|
| 14:40           |`<null>`|
| 14:41           |`<null>`|
| 14:42           |`<null>`|
| 14:43           |`<null>`|
| 14:44           |`<null>`|
| 14:45           |`<null>`|

** Workaround **
Modify the SQL Query to use a Case statement and return 0 instead of null:
```
  CASE 
  	WHEN (intervals.counted + truncated_recent_leads.counted) IS NULL THEN 0
  	ELSE (intervals.counted + truncated_recent_leads.counted)
  END AS 'counted'
```

**Server**

MySql8, `OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_232-b09)`, Ubuntu 18.04

Open Source work almost always goes unthanked. Thank you very much for your work, this piece of software is incredible. Please consider adding a "pay what you want" button somewhere as an option to help fund the project.

**Comment by flamber** [comment]
Hi @exabrial 

Took a little time to understand this completely. It doesn't blow up anything, so giving it P2 for now, since it seems now easy to come in this situation, but please let me know if it might have a more negative impact.

**Steps to reproduce:**
1. Simple question > Sample Dataset > Orders
2. Summarized by "Sum of Discount", grouped by Quantity
3. Change visualization to Bar
4. Filtered by Quantity between 16 and 20, so it's only null values:
![image](https://user-images.githubusercontent.com/1447303/78168514-5a55ce80-7450-11ea-9fd6-0dce50070aff.png)
5. Try changing the filter between 16 and 21, so it includes a single non-null value, then it works:
![image](https://user-images.githubusercontent.com/1447303/78168641-92f5a800-7450-11ea-9a35-eff9a9196908.png)

**Comment by flamber** [comment]
This was fixed in 0.35.3 - likely the same fix as 12251
